pallets/rmrk-core/src/lib.rs

```diff
@@ -28,11 +28,16 @@ pub type InstanceInfoOf<T> = NftInfo<
 	<T as frame_system::Config>::AccountId,
 	BoundedVec<u8, <T as pallet_uniques::Config>::StringLimit>,
 >;
+pub type ResourceOf<T, R, P> = ResourceInfo::<
+	BoundedVec<u8, R>,
+	BoundedVec<u8, <T as pallet_uniques::Config>::StringLimit>,
+	BoundedVec<PartId, P>
+	>;
 
 pub type BoundedCollectionSymbolOf<T> = BoundedVec<u8, <T as Config>::CollectionSymbolLimit>;
 
-pub type ResourceOf<T, R> =
-	ResourceInfo<BoundedVec<u8, R>, BoundedVec<u8, <T as pallet_uniques::Config>::StringLimit>>;
+// pub type ResourceOf<T, R> =
```

Comment:
We can remove the unused code, right?

Reply:
fab8c5e569c187e450c20120952bf6794bf8d169